### DISCUSSION TITLE
v0.0.15

### DISCUSSION THREAD

**Discussion by mkckr0** [mkckr0/audio-share#30] [Announcements]
- This version is all about Server MFC.
- Set `192.168.x.x` as default when exists multiple host IP #29.
- Add `config.ini` config file. It will save the host, port and other states.
- Add a `/hide` option so that the server app can start sliently.
- Add a "Auto Run" checkbox. If you check this, the server app will run with `/hide` at OS boot #29.

<hr /><em>This discussion was created from the release <a href='https://github.com/mkckr0/audio-share/releases/tag/v0.0.15'>v0.0.15</a>.</em>

**Comment by ghost** [comment]
HI, i what to run the server but i dont find the way can you help me? Ive extracted the tar.gz and now I dont know what to do, thx!

**Reply by mkckr0** [reply]
Follow the instruction of https://github.com/mkckr0/audio-share?tab=readme-ov-file#usage-for-windowslinux-cmd

**Reply by ghost** [reply]
Thx for your reply, I asked an ai and it gave me the answer, I was confused with the as-cmd -b xxx.xxx.xxx.xxx command 😉

**Reply by mkckr0** [reply]
`xxx.xxx.xxx.xxx` is your PC LAN addresss. Your phone and PC must be connected in a same LAN.
